modules/download-network-baselines.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// * operating/manage-network-policies.adoc
+:_content-type: PROCEDURE
+[id="download-network-baselines_{context}"]
+= Downloading network baselines from the network graph (2.0 preview)
+
+You can download network baselines from the network graph view.
+
+.Procedure
+. In the {product-title-short} web portal, navigate to *Network Graph (2.0 preview)*.
+. Click the *Namespace* list and use the search field to locate a namespace or select individual namespaces.
+. Click the *Deployments* list and use the search field to locate a deployment or select individual deployments to display in the network graph.
+. In the network graph, click on a deployment to view the information panel.
+. The *Baseline* tab lists the baseline flows. Use the *filter by entity name* field to further restrict the list of flows.
+. Optional: Check the box to exclude ports and protocols.
+. Click *Download baseline as network policy* to download a YAML file for the baseline.
```

Comment:
```suggestion
. Click *Download baseline as network policy*.
```

Reply:
I feel like it's important to tell them it will be a YAML file? Maybe this is obvious, though.

Reply:
Rewrote.